### PR TITLE
Fix inputs table format

### PR DESCRIPTION
**Pull request by aaron-lane**

### CODE REVIEW

File: modules/gsuite_enabled/variables.tf [hunk 1]
```diff
@@ -152,7 +152,7 @@ variable "disable_services_on_destroy" {
 }
 
 variable "default_service_account" {
-  description = "Project default service account setting: (delete | depriviledge | keep)"
+  description = "Project default service account setting: delete; depriviledge; keep."
```
Comment (Jberlinsky) [inline modules/gsuite_enabled/variables.tf]: Thoughts on spelling this out, e.g. "Project default service account setting. Can be one of `delete`, `depriviledge`, or `keep`."

File: variables.tf [hunk 2]
```diff
@@ -137,7 +137,7 @@ variable "disable_services_on_destroy" {
 }
 
 variable "default_service_account" {
-  description = "Project default service account setting: (delete | depriviledge | keep)"
+  description = "Project default service account setting: delete; depriviledge; keep."
```
Comment (Jberlinsky) [inline variables.tf]: Thoughts on spelling this out, e.g. "Project default service account setting. Can be one of `delete`, `depriviledge`, or `keep`."

Comment (morgante) [reply variables.tf]:
```suggestion
  description = "Project default service account setting: can be one of `delete, `depriviledge`, or `keep`."
```

File: modules/gsuite_enabled/variables.tf (hunk 1 above)
Comment (morgante) [reply modules/gsuite_enabled/variables.tf]:
```suggestion
  description = "Project default service account setting: can be one of `delete, `depriviledge`, or `keep`."
```

**Pull request merged**